docs/source/release_notes/0_32_0.rst

```diff
@@ -1,13 +1,10 @@
-===================
-0.32.0: Road Runner
-===================
+====================
+0.32.0 (Road Runner)
```

Comment:
Honestly I'd go with `0.32.0 Road Runner`, `0.32.0 - Road Runner`, `0.32.0: Road Runner` or `0.32.0 AKA Road Runner`. I don't like the brackets, though.